content/documentation/pages/1-installation/1-local/2-docker-customize.md

```diff
@@ -0,0 +1,200 @@
+
+3.  Replace the following:
+
+            depends_on:
+              - kafka
+
+    With:
+
+            depends_on:
+              - rabbitmq
+
+4.  Modify the `app-import` service definition `command` attribute to
+    replace
+    `https://bit.ly/Einstein-SR2-stream-applications-kafka-maven` with
+    `https://bit.ly/Einstein-SR2-stream-applications-rabbit-maven`.
+
+## Using InfluxDB Instead of Prometheus
+
+You can use InfluxDB rather than Prometheus for monitoring time-series
+database. To do so:
+
+1.  Replace the following configuration under the `services` section:
+
+          prometheus:
+            image: springcloud/spring-cloud-dataflow-prometheus-local:${DATAFLOW_VERSION:?DATAFLOW_VERSION is not set! Use 'export DATAFLOW_VERSION=local-server-image-tag'}
+            container_name: 'prometheus'
+            volumes:
+              - 'scdf-targets:/etc/prometheus/'
+            ports:
+              - '9090:9090'
+            depends_on:
+              - service-discovery
+
+          service-discovery:
+            image: springcloud/spring-cloud-dataflow-prometheus-service-discovery:0.0.3
+            container_name: 'service-discovery'
+            volumes:
+              - 'scdf-targets:/tmp/scdf-targets/'
+            expose:
+              - '8181'
+            ports:
+              - '8181:8181'
+            environment:
+              - metrics.prometheus.target.refresh.cron=0/20 * * * * *
+              - metrics.prometheus.target.discovery.url=http://localhost:9393/runtime/apps
+              - metrics.prometheus.target.file.path=/tmp/targets.json
+            depends_on:
+              - dataflow-server
+
+    With the following:
+
+          influxdb:
+            image: influxdb:1.7.4
+            container_name: 'influxdb'
+            ports:
+              - '8086:8086'
+
+2.  In the `dataflow-server` services configuration block, replace the
+    following `environment` entries:
+
+              - spring.cloud.dataflow.applicationProperties.stream.management.metrics.export.prometheus.enabled=true
+              - spring.cloud.dataflow.applicationProperties.stream.spring.cloud.streamapp.security.enabled=false
+              - spring.cloud.dataflow.applicationProperties.stream.management.endpoints.web.exposure.include=prometheus,info,health
+
+    With the following:
+
+              - spring.cloud.dataflow.applicationProperties.stream.management.metrics.export.influx.enabled=true
+              - spring.cloud.dataflow.applicationProperties.stream.management.metrics.export.influx.db=myinfluxdb
+              - spring.cloud.dataflow.applicationProperties.stream.management.metrics.export.influx.uri=http://influxdb:8086
+
+3.  Modify the `grafana` service definition `image` attribute to replace
+    `spring-cloud-dataflow-grafana-prometheus` with
+    `spring-cloud-dataflow-grafana-influxdb`.
+
+## Accessing the host file system
```

Comment:
For discussion.  In this section should we focus on mounting the .m2 since our examples lead us that direction.   Since to my knowledge we only show maven and docker resources and not file.